react/features/base/conference/reducer.js

```diff
@@ -209,7 +209,10 @@ function _lockStateChanged(state, action) {
         return state;
     }
 
-    return setStateProperty(state, 'locked', action.locked || undefined);
+    return setStateProperties(state, {
+        locked: action.locked || undefined,
+        lockPassword: action.locked ? state.lockPassword : null
```

Comment:
@virtuacoplenny, I'm confused by the introduction of `lockPassword` in the (redux) state of the conference alongside the existing `password`.

1. Generally, there's one password in a conference in the context of `conference.join` & `conference.lock` - the password one uses with `conference.lock` is what others will have to use with `conference.join`. If `conference.join` or `conference.lock` complete successfully, the password they completed successfully with should become `password`.

2. Depending on the implementation, I can image one may need a "temporary password" from between the time the user specifies a password to be utilized for `conference.lock` and the time of the successful completion of `conference.lock`. But once `conference.lock` completes successfully, that "temporary password" should surely turn into `password`. I don't expect such a "temporary password" to necessarily have to be a (redux) state of the password, I'd rather expect it to be a state of some sort of some other, probably UI-related feature. Anyway, (1) I don't think that's your idea of `lockPassword` and (2) even if that was your idea, at the every least some non-trivial jsdoc comments would've been necessary.

Could you please remove `lockPassword` in order to bring clarity?